spec/mailers/admin_notice_mailer_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+describe AdminNoticeMailer do
```

Comment:
These are good tests that cover all the important stuff 👍

I think we should use the `let` pattern, rather than this pattern of assigning instance variables here. Checkout spec/mailers/reply_notifier_spec.rb for an example of this.